.github/workflows/issue-verification.yml

```
@@ -0,0 +1,26 @@
+    - cron: '0 * * * *'
+  workflow_dispatch:
+
+jobs:
+  verify-issues:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
```

actionlint (1.7.4)

the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)